manifest.yml

```diff
@@ -1,7 +1,7 @@
 ---
 declared-services:
   sms_nlu_service:
-    label: natural-language-understanding
+    label: sms_nlu_service
```

Comment:
The service label comes from the Watson service.  I don't think it should be changed.